File: 2.0/csharp/calls/getCallRecordingByName.cs
using CallfireApiClient;
using CallfireApiClient.Api.Campaigns.Model;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        CallRecording recording = client.CallsApi.GetCallRecordingByName(379506003, "call-recording1", "callId,campaignId,lengthInBytes,mp3Url");
    }
}

File: 2.0/java/calls/getCallRecordingByName.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.campaigns.model.CallRecording;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api_login", "api_password");
        CallRecording recording = client.callsApi().getCallRecordingByName(379506003L, "call-recording1", "callId,campaignId,lengthInBytes,mp3Url");
    }
}
